packages/material-ui-lab/src/Autocomplete/Autocomplete.js

```diff
@@ -580,11 +580,11 @@ Autocomplete.propTypes = {
   /**
    * The default input value. Use when the component is not controlled.
    */
-  defaultValue: PropTypes.oneOfType([PropTypes.any, PropTypes.array]),
+  defaultValue: PropTypes.any,
   /**
    * If `true`, the input can't be cleared.
    */
-  disableClearable: PropTypes.bool,
+  disableClearable: PropTypes.any,
```

Comment:
This is because mapped type is used in the interface. Is there a way to hint proptype generator which type to use?

Reply:
Yes, there is a `/* @typescript-to-proptypes-ignore */` flag that can be used